### ISSUE TITLE
Query on processing of DocILE data

### ISSUE THREAD

**Issue by ziyangliu666** [nttmdlab-nlp/InstructDoc#4]
https://github.com/nttmdlab-nlp/InstructDoc/blob/c23c13631362ec7c3087a49b532ed094c21e782b/data_preprocessors/docile.py#L44

In the original DocILE repository, we couldn't find any instructions on how to generate JPG images for the dataset. We would appreciate it if you could provide information on how your team generates images for this dataset. Additionally, we would like to confirm whether the file `0001-1.jpg` represents the first page of the corresponding PDF document.

We observed that only the image of the first page is utilized, while the OCR is retained for all pages. This approach may introduce inconsistencies between the image and the OCR data. Could you please clarify this inconsistency and provide any relevant details?

Thank you for your assistance.

**Comment by rtanaka-lab** [comment]
We used [pdf2image](https://pypi.org/project/pdf2image/) to convert PDF into JPG images. `x` in the file `0001-x.jpg` denotes the page number of the corresponding PDF document (e.g., 1 represents the first page).

For all datasets during training, we used annotations of only the first pages. This is because we want to evaluate under zero-shot setting (Cross Domain setting), where evaluating models over multi-page documents that were not used during training.
